lib/f_core/device/c_rtc.cpp

```diff
@@ -9,13 +9,20 @@ LOG_MODULE_REGISTER(CRtc);
 
 CRtc::CRtc(const device& dev) : rtc(dev) {
     rtc_time rtcTime{0};
-#ifdef CONFIG_RTC_STM32
```

Comment:
should this be something slight different bc you need to special case the stm rtc but still need to set the year for other rtcs? did this just get left form testing on this hw?

Reply:
No. Turns out that this is the same case for the external RTC we use where it'll default to 2000.  I think we should just simplify and default to 100 for simplicity because of this. Knowing Launch, I presume they'll just copy over the RTC schematic if they use it in other boards as long as it works. We can address special cases if it comes up if we support some other hardware platform and it happens to support years before 2000.

Reply:
oh interesiting. neat